maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/IntegrationTestMojo.java

```diff
@@ -476,10 +476,16 @@
     @Parameter( property = "failsafe.systemPropertiesFile" )
     private File systemPropertiesFile;
 
-    @Parameter( property = "junitIncludeEngine" )
+    /**
+     * Provide the ID/s of an JUnit engine to be included in the test run.
```

Comment:
This looks much better now. The only problem is the version we are missing. We should say that this property is available since the version 3.0.0-M6. And the system property `junitincludeengine` is reserved keyword.

Similar with the next one. Thx

Reply:
Hmm I did not found example with marking some keyword as reserved system property description
Can you provide some example?

Reply:
We have very similar problem with `listener` in `testng.apt.vm`.
You should think of users and us, because if we do not mention that, the users will report a bug against us that the Provider fails. They would not notice that we also use the same property.

Reply:
I agree with you :) but still I am learning ways of working in your community and not all is know for me :)

Reply:
Ok, now I know which properties do you have in mind :)
Not @Parameters but system properties used internally to provide values to JUnitPlatformProvider.
Is it ok now?